package.json

```diff
@@ -19,7 +19,7 @@
   },
   "homepage": "https://github.com/Kaholo/kaholo-plugin-template#readme",
```

Comment:
Please replace all `kaholo-plugin-template` references to `kaholo-plugin-json`

Reply:
I forgot to push the changes.